js/data.js

```diff
@@ -1 +1,24 @@
 /* exported data */
+
+var $parent = document.querySelector('select');
+
+function getGhibliMovies() {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://ghibliapi.herokuapp.com/films');
+  xhr.responseType = 'json';
+  var titleArray = [];
+  xhr.addEventListener('load', function () {
+    for (var i = 0; i < xhr.response.length; i++) {
+      titleArray.push(xhr.response[i].title);
+      titleArray.sort();
+    }
+    for (var j = 0; j < titleArray.length; j++) {
+      var $option = document.createElement('option');
+      $option.textContent = titleArray[j];
+      $parent.appendChild($option);
+    }
+  });
+  xhr.send();
+}
+
+getGhibliMovies();
```

Comment:
You have this code in `main.js` already, that's where it should stay. You will end up using `data.js` for your data model and localStorage as the need arises